### PR TITLE
Remove Pickle for YAML

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

Pickle module appears to be in use, causing a possible security issue. It's not secure against erroneous or maliciously constructed data. It's recommended to never unpickle data received from an untrusted or unauthenticated source.

What component(s) does this PR affect?

- [x] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [x] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by slowr** [comment]
This code will cause new hijacks for the already running ones. We need to make sure the users that run this release after they have no active hijacks.

**Comment by pgigis** [comment]
Requires python migration script.

**Comment by vkotronis** [comment]
@slowr please resolve the conflicts here

**Comment by vkotronis** [comment]
Will re-check PR when the python migration script is ready

**Comment by vkotronis** [comment]
@slowr could you resolve the conflicts here? Besides these, the PR is tested right (i.e., are there any further actions needed from your side)? Do not merge yet, until @pgigis has finished the migration script.

**Comment by vkotronis** [comment]
@pgigis will re-check when you also recalculate the indexes (needs an extra sql script)

File: backend/migrate/migrate.py [hunk 1]
```diff
@@ -71,6 +68,52 @@ def migrate(next_db_version, db_cur, db_conn):
...
+        )
+        exit(-1)
+
+
+def migrate(next_db_version, db_cur, db_conn):
+    print("Executing migration {}...".format(next_db_version["db_version"]))
+    print("{0}".format(next_db_version["description"]))
+
+    if isinstance(next_db_version["file"], list):
+        for file in next_db_version["file"]:
```
Comment (vkotronis) [inline backend/migrate/migrate.py]: file is keyword in python, please use sth else

**Pull request merged**